data/json/npcs/epilogues.json

```diff
@@ -156,6 +156,174 @@
+  },
+  {
+    "type": "epilogue",
+    "group": "male",
+    "id": "npc_male_33",
+    "text": "     He found faith after your death, eventually joining an Anglican community and becoming a Vicar.  The crucifix provided him with a sense of meaning, before dying horribly at the hands of raiders.  He was still praying when they roasted him alive."
+  },
+  {
+    "type": "epilogue",
+    "group": "male",
+    "id": "npc_male_34",
+    "text": "     Lost and damned, he joined a bloody cult after your death, kidnapping and sacrificing young women.  His entire group was killed after being discovered by the Old Guard.  His head was put on a spike as a reminder that cultists will not be tolerated."
+  },
+  {
+    "type": "epilogue",
+    "group": "male",
+    "id": "npc_male_35",
+    "text": "     He did not survive for long after your death.  Electrocuted in an old cellar, he kept screaming your name.  His body was never found."
+  },
+  {
+    "type": "epilogue",
+    "group": "male",
+    "id": "npc_male_36",
+    "text": "     Survived you by just a few days, he drowned trying to escape from a horde."
+  },
+  {
+    "type": "epilogue",
+    "group": "male",
+    "id": "npc_male_37",
+    "text": "     He wandered alone for days, before finding a Free Merchant outpost.  He spent a few years as a caravan guard, often drinking himself half blind and enjoying any woman he could find.  Booze made his grip unsteady and he was eventually fired.  He died of alcohol poisoning the same day."
+  },
+  {
+    "type": "epilogue",
+    "group": "male",
+    "id": "npc_male_38",
+    "text": "     Obsessed with the idea of settling down, he eventually found love and retired to a farm in the middle of nowhere.  Relatively safe from the monsters, he thrived for years. Survived by seven sons and two daughters."
+  },
+  {
+    "type": "epilogue",
+    "group": "male",
+    "id": "npc_male_39",
+    "text": "     Became famous in the wastes after finding an old brewery and copying their recipe.  His chilled beer, a known delicacy, is traded far and wide by the Free Merchants."
+  },
+  {
+    "type": "epilogue",
+    "group": "male",
+    "id": "npc_male_40",
+    "text": "     Found a whole cellar of single malt whisky. Drank himself to death."
+  },
+  {
+    "type": "epilogue",
+    "group": "male",
+    "id": "npc_male_41",
+    "text": "     Depressed after your death, he would hang himself after just a few days."
+  },
+  {
+    "type": "epilogue",
+    "group": "male",
+    "id": "npc_male_42",
+    "text": "     Wandered for months, before finding a small community that would accept him.  Became a farmer and spent the rest of his uneventful life tending to crops, glad that he was no longer alone."
+  },
+  {
+    "type": "epilogue",
+    "group": "male",
+    "id": "npc_male_43",
+    "text": "     Obsessed with finding the cure, he died a few weeks later in an old lab, torn to shreds by a security turret."
```

Comment:
Cure to what?

Reply:
I thought maybe it's the cure to some pathogen he was infected with. 
Maybe use "the Cure" instead?

Reply:
Sounds good, I'll update it.